Bugs noticed:
	
	: stuff 1 2 . . CR ;
	: things stuff stuff ; //prints 2 1
	: stuff 4 5 . . CR ;
	things //still prints 2 1 even though stuff was updated
	
	(the online repl.it does this too, might be a part of the language...)

exception mechanism works...? not tested.

use clone() to spawn forth interpreters in seperate threads!
( really cheap to create and destroy! )

we should allocate memory dynamically for this to work well

